src/modules/home/heroSlider.tsx

```diff
@@ -11,10 +11,10 @@ import {
 import Image from "next/image";
 
 const heroImages = [
-  { id: 1, imageSrc: "/images/hero-image.jpg", imageAlt: "group image 1" },
-  { id: 2, imageSrc: "/images/hero-image.jpg", imageAlt: "group image 2" },
-  { id: 3, imageSrc: "/images/hero-image.jpg", imageAlt: "group image 3" },
-  { id: 4, imageSrc: "/images/hero-image.jpg", imageAlt: "group image 4" },
+  { id: 1, imageSrc: "/images/hero-image-1.jpg", imageAlt: "group image 1" },
+  { id: 2, imageSrc: "/images/hero-image-2.jpg", imageAlt: "group image 2" },
+  { id: 3, imageSrc: "/images/hero-image-3.jpg", imageAlt: "group image 3" },
```

Comment:
Can we also please reduced the size ? **1.28MB** seems too big.

Reply:
@thorn-pod thanks for your input. I have converted the image format to .webp to reduce the file size in the latest commit.